bin/options.js

```diff
@@ -22,9 +22,10 @@ function getOptions() {
   try {
     var opts = fs.readFileSync(optsPath, 'utf8')
           .trim()
+          .replace(/\\\s+/, '%20')
           .split(/\s+/)
-          .filter(function(value) {
-            return value ? true : false;
```

Comment:
I was thinking (incorrectly) that the `map` would do the same as the `filter` so it wasn't needed. 

That said, I'm not sure how useful `filter` is here. We've already trimmed everything and split against multiple spaces so I don't see a case where `value` would be `false` or `undefined`. What am I missing?